srai/regionizers/_spherical_voronoi.py

```diff
@@ -20,6 +20,8 @@
 from tqdm import tqdm
 from tqdm.contrib.concurrent import process_map
 
+CPU_COUNT = cpu_count()
```

Comment:
I'm not sure if this matters, but is it okay for this to be defined on import? It just seems to me it should be accessed on usage and not on module import.
I'm not sure if it has any significant implications though

Reply:
good catch, resolved